app/extras/app_config.rb

```diff
@@ -27,7 +28,9 @@ def self.theme
       primary_palette:                   ENV.fetch('THEME_PRIMARY_PALETTE', 'orange'),
       accent_palette:                    ENV.fetch('THEME_ACCENT_PALETTE', 'cyan'),
       primary_palette_config: JSON.parse(ENV.fetch('THEME_PRIMARY_PALETTE_CONFIG', '{"default": "400"}')),
-      accent_palette_config:  JSON.parse(ENV.fetch('THEME_ACCENT_PALETTE_CONFIG', '{"default": "500"}'))
+      accent_palette_config:  JSON.parse(ENV.fetch('THEME_ACCENT_PALETTE_CONFIG', '{"default": "500"}')),
+      custom_primary_palette: json_parse_or_false('THEME_CUSTOM_PRIMARY_PALETTE'),
```

Comment:
What's the difference between having a custom primary palette, and defining the palette you want in THEME_PRIMARY_PALETTE_CONFIG? Not sure I see the need for these new ENVs yet

Reply:
The client does not want any of the material palettes, so I have to generate one (two actually) and pop it in as a new palette.